Standards/scs-0106-v1-evaluate-current-state-of-cloudmon.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Evaluating the deployment of CloudMon in SCS infrastructure
+type: Decision Record
+---
+
+## Introduction
+
+In the fast-paced environment of modern cloud computing, ensuring the reliability and performance
+of cloud infrastructure is paramount for organizations. Effective monitoring and testing framework
+are essential tools in this endeavor, enabling proactive identification and resolution of issues before
+they impact critical operations. However, selecting the right monitoring solution can be a daunting task,
+particularly when faced with options that are overly complex and lack essential documentation and working
+scenarios.
+
+## Motivation
+
+One such solution, the [Cloudmon](https://stackmon.org/) project, faces challenges that might limit its effectiveness for organizations looking
+for efficient and reliable monitoring capabilities. This introduction outlines the reasons why our organization opts
+against utilizing the Cloudmon project and instead embraces a more streamlined and effective approach involving Gherkin
+test scenarios and mapping Python API calls to interact with OpenStack resources. By addressing the complexities and
+shortcomings of the Cloudmon project, SCS organization aims to adopt a monitoring solution that not only meets but
+exceeds our requirements for simplicity, reliability, and ease of use.
+
+## Design Considerations
+
+Our approach was to base on a technical concept. This document serves as a proposal, with the final decision
+subject to discussion with SCS team members. We propose a behavior-driven system based on solutions using Java framework
```

Comment:
There are many different testing frameworks and testing strategies, which are not really differentiated in this text. You propose a behavior-driven test concept using cucumber, but somehow the document fails to mention any up- or downsides of behavior driven testing or of this specific framework.

There is no reasoning I have read in this text why behavior driven testing is superior to the cloudmon approach.

And I don't doubt that there might be advantages to it, but those need to be clearly spelled out somewhere.

This text also talks about a "technical concept", where can I see that? It isn't linked here, or included. If it is the basis for the decision, it surely should be included?

I only know both technologies superficially, for the record, that is cloudmon and cucumber.